snowplow-tracker/build.gradle

```diff
@@ -61,6 +61,7 @@ android {
 }
 
 dependencies {
+    implementation 'com.android.support:support-annotations:28.0.0'
```

Comment:
Just curious - what does this do?

Reply:
It lets us to add annotations to methods informing of `Nullable` and `NonNull` parameters.
I've just used it here:
```
public TrackerBuilder gdprContext(@NonNull Basis basisForProcessing, String documentId, String documentVersion, String documentDescription)
```
It helps because Android Studio raises a Warning if that param is set to null and it helps a lot when the developer uses Kotlin.